iocore/cache/P_CacheHosting.h

```diff
@@ -152,7 +152,7 @@ class CacheHostTable
 struct CacheHostTableConfig;
 typedef int (CacheHostTableConfig::*CacheHostTabHandler)(int, void *);
 struct CacheHostTableConfig : public Continuation {
-  CacheHostTable **ppt;
+  std::atomic<CacheHostTable **> ppt;
```

Comment:
I noticed this isn't used by anything external. It'd be nice to make it `private`, to enforce that, and make it easier for readers to know that